tests/README.md

```diff
@@ -1,57 +1,37 @@
-@page testing Testing Guide
+# Testing Guide
 
-[TOC]
+[Integration testing](https://en.wikipedia.org/wiki/Integration_testing) is an essential part of software development. It is performed when integrating code changes into the main development branch to verify that the code works as expected. The following sections describe how to run and add integration tests used to the svFSIplus program.
 
-[Integration testing](https://en.wikipedia.org/wiki/Integration_testing) is an essential part of software development. It is performed when integrating code changes into the main development branch to verify that the code works as expected. Below is a quick guide on how to run and add integration tests for `svFSI`.
+Running a test case requires 
+- Build svFSIplus
+- Install Git LFS used to download test data
+- Build svZeroDSolver (only required from certain tests)
 
+# Build svFSIplus
+svFSIplus can be built follow these [instructions](../README.md).
```

Comment:
by following these steps

Reply:
I changed it to `svFSIplus can be built following these [instructions](../README.md).`, the build process includes installing software and then the actual build steps.